src/styles/_blocks/_header.scss

```
@@ -0,0 +1,70 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  background: #fff;
+
+  &__logo {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin-left: 50px;
+  }
+
+  &__image {
+    width: 40px;
+    height: 40px;
+  }
+
+  &__nav {
+    display: flex;
+    margin-right: 50px;
+  }
+
+  &__list {
+    display: flex;
+    flex-direction: row;
```

its default and can be removed
```suggestion
    flex-direction: row;
```